views/index.erb

```diff
@@ -20,10 +20,15 @@
 
 		<b><%= e.type.capitalize %>:</b> <%= e.message %>
 
-		</div>	
+		</div>
 	<% end %>
 <% end %>
 
+<% if @url.length > 0 %>
+<p>GET validation response in <a href="<%= request.fullpath + "&format=json" %>">JSON format</a>.</p>
```

Comment:
On the (HTML) result page, linking to the corresponding result page in JSON format, to make this feature more discoverable.